### PR TITLE
[INLONG-447] Standardized Version Variables

### PR DESCRIPTION
**Pull request by bluewang**
Fixes https://github.com/apache/inlong-website/issues/447

where *XYZ* should be replaced by the actual issue number.

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/version.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const siteVariables = {
+  inLongVersion: '1.3.0-SNAPSHOT[[[[[[',
```
Comment (leezng) [inline docs/version.js:2]: is it ok?

File: versioned_docs/version-1.2.0/version.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const siteVariables = {
+  inLongVersion: '1.2.0-incubating}}}}}}}}',
```
Comment (leezng) [inline versioned_docs/version-1.2.0/version.js:2]: as above

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/oracle.md [hunk 3]
```diff
@@ -15,18 +17,18 @@ sidebar_position: 12
 
 ## 依赖
 
-为了设置 Oracle Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 SBT）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
+为了设置 Oracle Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 version）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/oracle.md:20]: You should not change `SBT` to `version`. `SBT` is a build tool like `maven`.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/postgresql.md [hunk 4]
```diff
@@ -16,18 +18,18 @@ sidebar_position: 13
 
 ## 依赖
 
-为了设置 PostgreSQL Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 SBT）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
+为了设置 PostgreSQL Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 version）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/postgresql.md:21]: You should not change SBT to version. SBT is a build tool like maven

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/sqlserver.md [hunk 5]
```diff
@@ -15,18 +17,18 @@ sidebar_position: 14
 
 ## 依赖
 
-为了设置 SQLServer Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 SBT）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
+为了设置 SQLServer Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 version）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/sqlserver.md:20]: You should not change SBT to version. SBT is a build tool like maven

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/tdsql-postgresql.md [hunk 6]
```diff
@@ -15,18 +17,18 @@ sidebar_position: 15
 
 ## 依赖
 
-为了设置 TDSQL-PostgreSQL Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 SBT）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
+为了设置 TDSQL-PostgreSQL Load 节点, 下面提供了使用构建自动化工具（例如 Maven 或 version）和带有 Sort Connector JAR 包的 SQL 客户端的两个项目的依赖关系信息。
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/tdsql-postgresql.md:20]: You should not change `SBT` to `version`. `SBT` is a build tool like `Maven`.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/version.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export const siteVariables = {
+  inLongVersion: '1.3.0-SNAPSHOT中c',
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/version.js]: Is it right?

**Review by yunqingmoswu — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**